en/backup-restore-faq.md

```diff
@@ -11,10 +11,88 @@ This document describes the common questions that occur during EBS snapshot back
 
 You might encounter the following problems during EBS snapshot backup:
 
+- [Backup failed after an upgrade of the TiDB cluster](#backup-failed-after-an-upgrade-of-the-tidb-cluster)
 - [Failed to start a backup or the backup failed immediately after it started](#failed-to-start-a-backup-or-the-backup-failed-immediately-after-it-started)
 - [The backup CR of a failed task could not be deleted](#the-backup-cr-of-a-failed-task-could-not-be-deleted)
 - [Backup failed](#backup-failed)
 
+### Backup failed after an upgrade of the TiDB cluster
+
+After the TiDB cluster is upgraded from an earlier version to v6.5.0, the backup using volume snapshots might fail with the following error:
+
+```shell
+error="min resolved ts not enabled"
+```
+
+The backup failed because the PD configuration `min-resolved-ts-persistence-interval` is set to 0, which means that the globally consistent min-resolved-ts service of PD is disabled. The EBS volume snapshot requires this service to obtain the globally consistent ts of the cluster. You can check this configuration using either of the following ways:
```

Comment:
```suggestion
The backup failed because the PD configuration `min-resolved-ts-persistence-interval` is set to 0, which means that the globally consistent min-resolved-ts service of PD is disabled. The EBS volume snapshot requires this service to obtain the globally consistent timestamp of the cluster. You can check this configuration using either of the following ways:
```